webapp/channels/src/components/suggestion/search_user_provider.tsx

```diff
@@ -70,7 +70,7 @@ export default class SearchUserProvider extends Provider {
     }
 
     handlePretextChanged(pretext: string, resultsCallback: ResultsCallback<UserProfile>) {
-        const captured = (/\bfrom:\s*(\S*)$/i).exec(pretext.toLowerCase());
+        const captured = (/\bfrom:\s*(\S*)$/i).exec(pretext.replace('@', 'from:').toLowerCase());
```

Comment:
Can you explain what this is supposed to do?

Reply:
when @ is typed, I replace it from 'from:' so to invoke function to show suggestion